scanners/android/android.go

```diff
@@ -107,6 +108,15 @@ func (scanner *Scanner) ExcludedScannerNames() []string {
 
 // Options ...
 func (scanner *Scanner) Options() (models.OptionModel, models.Warnings, error) {
+
+	// Search for local.properties file
+	for _, filePath := range scanner.FileList {
+		if strings.Contains(filePath, "local.properties") {
+			log.Warnft(`the local.properties file should not be committed into the repository. The location of the file is:
```

Comment:
Question: Does it should be in the `warnings := models.Warnings{}` model, as we want to display the warning message on the UI as well, not only in the scanner's log?